composer.json

```
@@ -4,7 +4,8 @@
     "description": "Flow Form Framework preset for Fusion based Form rendering",
     "license": "MIT",
     "require": {
-        "neos/form": "^5.0"
+        "neos/form": "^5.0",
+        "neos/fusion": ">=7.3"
```

why didn't you use semantic version constraints, e.g. (`^7.3`)?

---

Yeah, than we need to change it with each Neos version.

---

And that makes sense! Never use unlimited constraints!
```suggestion
        "neos/fusion": "^7.3 || ^8.3 || ^9.0"
```

---

```suggestion
        "neos/fusion": "^7.3 || ^8.0 || ^9.0"
```

"^8.0" right?

---

🙈 Yes, of course!